docs/features/global_tools.md

```diff
@@ -113,6 +113,25 @@ exposed = { ansible = "ansible" } # (1)!
 
 1. The `ansible` binary is exposed even though it is installed by a dependency of `ansible`, the `ansible-core` package.
 
+It's also possible to expose an executable which is located in a nested directory.
+For example dotnet.exe executable is located in a dotnet folder,
+to expose dotnet.exe you must specify its relative path :
+
+```
+pixi global install dotnet --expose dotnet=dotnet\dotnet.exe
+```
+you can also omit the extension
+```
+pixi global install dotnet --expose dotnet=dotnet\dotnet
```

Comment:
I tried this example on Linux but that package doesn't seem to work. Do you understand why the `dotnet` setup is using a non `bin/xx` setup for it's binaries?

Reply:
I have no Idea.
I had never tried to install dotnet on linux but your right it's installed in `.pixi/envs/dotnet/lib/dotnet/dotnet`
For some reason everything is painful with windows tools...
Maybe a more cross platform or generic tool would be better for the documentation.

Reply:
Yeah, I contacted @dhirschfeld as he is a pixi user and the maintainer of `dotnet` maybe he has some smart things to say about this.

Reply:
Not sure about smart - the TL;DR is that it's just a binary repackage of upstream and that seems to be the way it's packaged by Microsoft.

The package sets a number of env vars in an activation script, including the PATH. I'd like to port setting/unsetting the env vars to the JSON format (assuming that works with `pixi`?) but haven't gotten around to it.

Reply:
Yes pixi supports that :+1:

Reply:
Is there another example package, where this would be required, that does work on linux?